tortoise/__init__.py

```diff
@@ -154,7 +154,7 @@ def split_reference(reference: str) -> Tuple[str, str]:
                     model._meta.db_table = model.__name__.lower()
 
                 # TODO: refactor to share logic between FK & O2O
-                for field in model._meta.fk_fields:
+                for field in sorted(model._meta.fk_fields):
```

Comment:
Why sorted?

Reply:
sorting here makes us have the FK fields always in the same order. Without this, the custom m2m sql generation presented inconsistent behavior regarding the generated foreign key fields.